modules/nlu/src/backend/engine.ts

```diff
@@ -119,14 +139,14 @@ export default class Engine implements NLUEngine {
     if (this.modelAlreadyLoaded(model)) {
       return
     }
-    if (!model.data.output) {
+    if (!model.data.intents) {
```

Comment:
I'm not sure you fully understand why this was here so let's make a quick debug session so we can clarify the implication of that